### PR TITLE
Fixes #23423 - Loosen seamless-immutable and lodash dependencies

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Review by ehelms — DISMISSED** [review]

**Comment by ekohl** [comment]
Looks like this fails the tests. Will look into this.

File: webpack/assets/javascripts/react_app/components/statistics/ChartBox.test.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ jest.unmock('../../../services/ChartService');
 jest.unmock('./ChartBox');
 
 describe('ChartBox', () => {
-  const setup = ({ status, chart = { id: 2 } }) => shallow(<ChartBox
+  const setup = ({ status, chart = { id: '2' } }) => shallow(<ChartBox
```
Comment (ekohl) [inline webpack/assets/javascripts/react_app/components/statistics/ChartBox.test.js:11]: This is an unrelated change, but I was annoyed by the warning it generated.

**Review by ekohl — COMMENTED** [review]
Updated to bump the dependency to ^7.1.2 to properly handle the requestError in the snapshot in a consistent way.

**Comment by ekohl** [comment]
Upgrade failed during git pull so I think that's unrelated.

**Review by sharvit — APPROVED** [review]
LGTM!

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**